src/helpers/ssoauthhelper.js

```diff
@@ -12,7 +12,7 @@ let retryGetAccessToken = 0;
 
 export async function getGraphData() {
   try {
-    let bootstrapToken = await OfficeRuntime.auth.getAccessToken({ allowSignInPrompt: true, forMSGraphAccess: true });
```

Comment:
this param was purposely removed because it causes Outlook SSO to fail on Desktop. Made this same change in SSO TS repo as well